Sources/PointFree/RunnerTasks/ValidateEnterpriseEmails.swift

```diff
@@ -0,0 +1,92 @@
+import Either
+import Mailgun
+import Models
+import PointFreePrelude
+import Prelude
+
+public func validateEnterpriseEmails() -> EitherIO<Error, Prelude.Unit> {
+
+  return Current.database.fetchEnterpriseEmails()
+    .flatMap(validate(enterpriseEmails:))
+    .flatMap(sendValidationSummaryEmail(results:))
+}
+
+private func validate(enterpriseEmails: [EnterpriseEmail]) -> EitherIO<Error, [ValidationResult]> {
+  return lift(
+    sequence(
+      enterpriseEmails
+        .map(validate(enterpriseEmail:))
```

Comment:
here we run a bunch of validations in parallel. kinda wish we had something like a `ParallelValidation` so that I could easily retain error info